contracts/token/IERC721_Receiver.cairo

```diff
@@ -12,4 +12,13 @@ namespace IERC721_Receiver:
         data: felt*
     ) -> (selector: felt): 
     end
+
+
+    # ERC721's `safeTransferFrom` requires a means of differentiating between account and
+    # non-account contracts. Currently, StarkNet does not support error handling from the
+    # contract level; therefore, this ERC721 implementation requires that all contracts that
+    # support safe ERC721 transfers (both accounts and non-accounts) include the `is_account` 
+    # method. This method should return `0` since it's NOT an account.
+    func is_account() -> (res: felt):
+    end
```

Comment:
I think this is a great reason to go with @spalladino's idea of using ERC165 instead of `is_account`. It would be a bit less weird than requiring a falsy `is_account` method to receive an NFT.